addons/common/config.cpp

```diff
@@ -14,6 +14,8 @@ class CfgPatches {
     };
 };
 
+class ctrlCombo;
```

Comment:
Should be moved into `Cfg3DEN.hpp`. This class will not be used by any other header file.

Reply:
done